webpack.config.js

```diff
@@ -29,7 +33,10 @@ module.exports = {
         test: /\.css$/,
         loaders: ['style', 'css'],
       },
+      {
+        test: /\.json$/,
+        loader: 'json-loader',
+      },
     ],
   },
-  plugins: [new webpack.BannerPlugin('@generated')],
```

Comment:
Why is this being removed? Is this an accident? It was originally added to prevent lint from validating the output static bundle which reports a ton of errors.

Reply:
This was a mistake! I'll add it back